### PR TITLE
Second Milestone: Works and About sections

### PR DESCRIPTION
**Pull request by Piercel2022**
For this second milestone, we will building 

- the works section with 4 differents cards containing the projects.
- the about myself section containing information about myself like languages, skills and frameworks
Works
About My Self

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,203 @@
...
+                  <ul class="categories_card4">
+                    <li><a href="#" class="ollinks">html</a></li>
+                      <li><a href="#" class="ollinks">css</a></li>
+                        <li><a href="#" class="ollinks">javascript</a></li>
+                  </ul></div>
+              <a class="button_card4" href="#">See Project</a>
+          </div>
+  
+   </section>
+   <section class="about_myself">
```
Comment (danielufeli) [inline index.html:140]: - [ ] Nice work you have done on the about section, kindly adjust the section width to full the page, so it aligns with what we have on the Figma Template

### Your Design
m1

### Figma Template
f1

**Review by danielufeli — CHANGES_REQUESTED** [review]
Hi @Piercel2022 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Clean Design ✔️
- Passing linters ✔️
- Descriptive Readme ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @danielufeli in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1,2 +1,52 @@
...
+A Linter for HTML & CSS
+
+
+
+
+## Author
+
+- GitHub: [@githubhandle](https://github.com/Piercel2022)
+- Twitter: [@twitterhandle](https://twitter.com/pier_celestin)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/piercelestinmax)
```
Comment (chrystalme) [inline README.md:31]: Hello @Piercel2022, nice to see that you have added your handles but you have not customises them to represent you. Kindly, update `@githubhandle` to `@Piercel2022`. Do the same for the rest handles.

**Review by chrystalme — CHANGES_REQUESTED** [review]
# Hi @Piercel2022, 

Great work on making the changes 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

## Highlights 💡 
- implemented changes suggested by previous reviewer ✔️  

## Suggested changes ♻️ 
_Check the comments under the review._

- Leaving git conflict resolution comments in your file makes your code base look unprofessional. Kindly fix the places where I have commented. Also, check that there are no such comments in your code base.

# Optional Requirement
- In this project, you are required to build using a mobile first approach. You do not have to worry how your page looks on other devices but your content should not be fixed just the mobile frame. This will help you while working on other device views.
Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Also, you can reach me on [Twitter](https://twitter.com/afam_ifeanyi) or [Slack](https://microverse-community.slack.com/team/U019ZC99KFA). Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Omar-Muhamad — APPROVED** [review]
# Approved ✔
### Hi @Piercel2022,

Great job, Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

thumbs-up-agree


## (Highlights) Good Points: :+1:
- You documented your work professionally. ✔
- Github flow is used correctly. ✔
- All linters checks are ok. ✔

## (Changes Required) Aspects to improve: ♻️

- N/A

## [Optional] suggestions:

- N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

**Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification**

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**